### PR TITLE
Added animation support to infantry stand sequences

### PR DESCRIPTION
**Pull request by Mailaender**
This is required for swim animations. https://github.com/OpenRA/ra2/pull/172 comes with a test case.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs [hunk 1]
```diff
@@ -74,7 +74,13 @@ public void PlayStandAnimation(Actor self)
 		{
 			var sequence = DefaultAnimation.GetRandomExistingSequence(Info.StandSequences, Game.CosmeticRandom);
 			if (sequence != null)
-				DefaultAnimation.PlayFetchIndex(NormalizeInfantrySequence(self, sequence), () => 0);
+			{
+				var normalized = NormalizeInfantrySequence(self, sequence);
+				if (DefaultAnimation.GetSequence(normalized).Length > 1)
+					DefaultAnimation.PlayRepeating(normalized);
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs]: `PlayRepeating` should work fine for the length-1 sequences too.

Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs]: You are right. Updated.

**Comment by obrakmann** [comment]
:+1:

**Comment by pchote** [comment]
:+1:

**Pull request merged**